### PR TITLE
docs: Add example of strategy-specific arguments in use

### PR DESCRIPTION
**Pull request by papistan**
@bmulholland referencing your comment on #991

### CODE REVIEW

File: docs/content/en/schemes/oauth2.md [hunk 1]
```diff
@@ -15,6 +15,12 @@ category: Schemes
 this.$auth.loginWith('social')
 ```
 
+Additional arguments can be passed through to Google as the `params` key of the second argument:
```
Comment (JoaoPedroAS51) [inline docs/content/en/schemes/oauth2.md:18]: Maybe here we can remove the word "Google", as this example refers to oauth2 scheme. What do you think?

> Additional arguments can be passed through as the `params` key of the second argument:

**Review by JoaoPedroAS51 — CHANGES_REQUESTED** [review]
Nice work! :) Just added a small note

File: docs/content/en/schemes/oauth2.md (hunk 1 above)
Comment (bmulholland) [reply docs/content/en/schemes/oauth2.md]: Good find. Changed!

**Review by JoaoPedroAS51 — APPROVED** [review]

**Comment by bmulholland** [comment]
Thanks @papistan !

**Comment by papistan** [comment]
@bmulholland @JoaoPedroAS51 - thanks for work getting this live!

**Pull request merged**